app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ReportsController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :new, :edit, :update, :destroy]
+
+  before_action :set_review, only: [:new, :create]
+  before_action :check_admin, only: [:index, :destroy]
```

Comment:
why dont we have it under admin namespace?

Reply:
Yes sir but I thought it could work with only one controller